### ISSUE TITLE
Select nearest video quality level when the preferred one is not available

### ISSUE THREAD

**Issue by dessant** [dessant/youtube-video-quality#21]
Currently the extension selects the highest available quality when the preferred one is not available. We should be able to instead configure the extension to select the nearest available quality that is higher or lower than the preferred one.

**Comment by simonlokk** [comment]
Yes, I have this issue as well. For example, I find 480p to be a good compromise of quality and bandwidth. However, while most YT videos have this option, not all do. In the case that the selected content does not have 480p, the extension will allow the HD stream to be played. Since the purpose of this extension is to cap resolution, I think the behavior in the event of a selected resolution being unavailable would be to select the next lowest resolution, which in this example would be 360p. Thanks for your time in reading this and in creating this useful extension.
